scripts/start_civ.sh

```diff
@@ -312,6 +312,18 @@ function set_pt_usb() {
     fi
 }
 
+function set_pt_udc() {
+    local UDC_PCI=$(lspci -D | grep -m2 "USB controller" | tail -1 | grep -o "....:..:..\..")
+    echo "passthrough USB device: $UDC_PCI"
+
+    if [[ $1 == "unset" ]]; then
+        set_pt_pci_vfio $UDC_PCI "unset"
+    else
+        set_pt_pci_vfio $UDC_PCI
+        GUEST_USB_PT_DEV=" -device vfio-pci,host=${UDC_PCI#*:},x-no-kvm-intx=on"
```

Comment:
Is there any scenario to passthrough both USB host and device controllers? If yes, please use another variable for GUEST_USB_PT_DEV, maybe GUEST_UDC_PT_DEV; if no, please describe in show_help(), only one of "--passthrough-pci-usb, --passthrough-pci-udc" can be specified when launch this script.

Reply:
Yes, let me correct this variable. There may be scenarios where we may require both.